assets/javascripts/discourse/components/modal/post-event-invitees/user.gjs

```diff
@@ -0,0 +1,30 @@
+import Component from "@glimmer/component";
```

Comment:
This is replacing the renderInvitee helper. We are actually rendering a User here.